src/graphql/typeDefs/dataTypes/eventThemeInput.graphql

```diff
@@ -0,0 +1,33 @@
+input EventThemeCreateInput {
+  "Path to hero image"
+  heroSlug: String
+
+  "Primary color of theme"
+  primary: String!
+
+  "Secondary color of theme"
+  secondary: String!
+
+  "Tertiary color of theme"
+  tertiary: String!
+
+  "Action on mouse-over of event?"
+  highlight: String!
+}
+
+input EventThemeUpdateInput {
```

Comment:
I don't think we want these all !.. that would mean we would need to update them all every time even if we just wanted to change 1.

Reply:
That is how you submitted them. I agree that updates are not all needed. I will update

Reply:
updated with commit cf4bfb72c7969d271fc7d2983b427d7a9a2de08a